lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_user.inc

```diff
@@ -39,6 +39,23 @@ function paraneue_dosomething_preprocess_user_profile(&$vars) {
         $impact = l($impact, 'reportback/' . $rbid);
       }
       $img = dosomething_image_get_themed_image_by_fid($reportback->fids[0], '300x300');
+
+      // Get all those kudos
+      $kudos_term_ids = dosomething_kudos_get_kudos_by_file($reportback->fids[0]);
+      if ($kudos_term_ids) {
+        $impact .= '<br/>';
+        foreach ($kudos_term_ids as $term_id) {
+
+          // Get the term name from the id and make it plural if necessary
+          $name = taxonomy_term_load($term_id->tid)->name;
```

Comment:
@angaither [here](https://github.com/DoSomething/phoenix/pull/6474/commits/fb8675184ca4a41f4cd2946783ca1cf9ce5084fc#diff-2af8eaa58f7c9cf6c95554e8bb26bdfeR261) it is!